comp/core/flare/flare.go

```diff
@@ -168,7 +174,11 @@ func (f *flare) Send(flarePath string, caseID string, email string, source helpe
 }
 
 // Create creates a new flare and returns the path to the final archive file.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It could be good to add to the function documentation that the `providerTimeoutSeconds` fallback to the configuration parameter when `providerTimeoutSeconds <= 0`

Reply:
Agree

Reply:
Done 👍